fixtures/stairville/kiinanledi-2-0.json

```diff
@@ -0,0 +1,102 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Kiinanledi 2.0",
```

Comment:
I can't find anything called "Kiinanledi" -- Google for this + Stairville (or DMX) turns up only 3 pages, all related to this PR.